File: JSONtoDB/src/es/iesnervion/nba/model/Team.java
package es.iesnervion.nba.model;
// city, abrebiacion, state, division

/**
 * @author apol
 *
 */
public class Team {
	
	private String full_name;
	private String city;
	private String abbreviation;
	private String state;
	private String division;
	
	public Team(String full_name, String city, String abrebiation, String state, String division) {
		super();
		this.full_name = full_name;
		this.city = city;
		this.abbreviation = abrebiation;
		this.state = state;
		this.division = division;
	}
	
	public String getFull_name() {
		return full_name;
	}
	
	public void setFull_name(String full_name) {
		this.full_name = full_name;
	}
	
	public String getCity() {
		return city;
	}
	
	public void setCity(String city) {
		this.city = city;
	}
	
	public String getAbrebiation() {
		return abbreviation;
	}
	
	public void setAbrebiation(String abrebiacion) {
		this.abbreviation = abrebiacion;
	}
	
	public String getState() {
		return state;
	}
	
	public void setState(String state) {
		this.state = state;
	}
	
	public String getDivision() {
		return division;
	}
	
	public void setDivision(String division) {
		this.division = division;
	}

	@Override
	public String toString() {
		return "Team [full_name=" + full_name + ", city=" + city + ", abbreviation=" + abbreviation + ", state=" + state
				+ ", division=" + division + "]";
	}
	
	
}

File: model/Team.php
<?php

/**
 * Created by PhpStorm.
 * User: apol
 * Date: 24/11/16
 * Time: 9:37
 */
class Team implements JsonSerializable
{
    private $codigo;
    private $full_name;
    private $city;
    private $abrebiacion;
    private $state;
    private $division;

    function __construct($codigo, $name, $city, $abrebiacion, $state, $division)
    {
        $this->codigo = $codigo;
        $this->full_name = $name;
        $this->city = $city;
        $this->abrebiacion = $abrebiacion;
        $this->state = $state;
        $this->division = $division;
    }

    function jsonSerialize()
    {
        return array(
            'codigo' => $this->codigo,
            'full_name' => $this->full_name,
            'city' => $this->city,
            'abrebiacon' => $this->abrebiacion,
            'state' => $this->state,
            'division' => $this->division
        );
    }

    function __sleep()
    {
        return array('codigo', 'full_name', 'city', 'abrebiacion', 'state', 'division');
    }

    /**
     * @return mixed
     */
    public function getFullName()
    {
        return $this->full_name;
    }

    /**
     * @return mixed
     */
    public function getCity()
    {
        return $this->city;
    }

    /**
     * @return mixed
     */
    public function getAbrebiacion()
    {
        return $this->abrebiacion;
    }

    /**
     * @return mixed
     */
    public function getState()
    {
        return $this->state;
    }

    /**
     * @return mixed
     */
    public function getDivision()
    {
        return $this->division;
    }




}